### DISCUSSION TITLE
MPB in Spokane, Washington

### DISCUSSION THREAD

**Discussion by MatthewPNWBTC** [MyFirstBitcoin/.github#22] [General]
Hello, my name is Matthew. I live in Spokane, Washington.  My brother and I recently started Pacific Northwest Bitcoin and currently offer some free classes, but we would love to replace those with offering the MPB diplomas and teaching this amazing curriculum.  We would teach this in English.

**Comment by RealMoneyBTC** [comment]
Hello Matthew...
Great to hear you are working on Bitcoin EDU in the PNW! We would love to support this effort.

**Reply by MatthewPNWBTC** [reply]
Thanks. Taught our first class tonight.  It was great.
